### PR TITLE
Update Brimcap docs for phase-out of "brimcap load" (for merge with brim/1716)

### PR DESCRIPTION
**Pull request by philrz**
This PR is part of the non-code coverage for #113.

Since the "a la carte" use of `brimcap analyze` and `brimcap index` already exists but just aren't used by the app yet, it was feasible for me to test and prep these updates in advance. Since it's written as if describing the current known behaviors of Brim, my intent is to hold off on merging this until we deliver the code changes expected in brimdata/brim#1716.

While I was in there, I corrected a couple other errors I spotted in the wiki articles and also added a couple analyzer caveats that @mattnibs recently shared with me. I adjusted the wording in a couple spots, so I'll add PR comments to highlight those and make sure they're confirmed before we commit.

### CODE REVIEW

File: docs/Custom-Brimcap-Config.md
Comment (philrz) [inline docs/Custom-Brimcap-Config.md:167]: @mattnibs: The way you originally described this caveat to me:

> It is assumed that if stdin is closed on a process the process will exit.

In that wording, the way the word "process" is used twice seemed ambiguous to me, since there's the analyzer process and the ones it invokes. I've tried to clarify that in my version, but please confirm that it's still accurate in my version.

File: docs/Custom-Brimcap-Config.md
Comment (philrz) [inline docs/Custom-Brimcap-Config.md:169]: @mattnibs: Once again, your original wording was:

> The process must ensure that all child processes that have inherited stdout/stderr have been killed before exiting.

I assume my slight rewording is still accurate, but, please confirm. Thanks!

File: docs/Custom-Brimcap-Config.md
Comment (philrz) [inline docs/Custom-Brimcap-Config.md:493]: This was a leftover from a previous version of the doc.

File: docs/Troubleshooting.md
Comment (nwt) [inline docs/Troubleshooting.md:32]:
```suggestion
invokes `brimcap search` to extract the packets for the target flow
```

**Review by nwt — APPROVED** [review]

File: docs/Custom-Brimcap-Config.md
Comment (philrz) [reply docs/Custom-Brimcap-Config.md:171]: Thanks @nwt. I'm happy to commit all your suggestions for this list of analyzer clarifications. I'd cc'ed @mattnibs  a couple times in these PR comments looking for clarifications on this same list since some of them were based on wording he provided, but I've not yet heard from him. Since I know you and he had a detailed Zoom session before his most recent round of Brimcap changes and you ultimately approved this PR, I'm assuming your wording improvements are based on a current understanding of the code, hence will merge.

File: docs/Custom-Brimcap-Config.md
Comment (philrz) [reply docs/Custom-Brimcap-Config.md]: Thanks @nwt. Indeed, your PR feedback in the Brim PR brought me around to setting `PATH` rather than that `$ZDEPS/` approach I was using, so I've pushed a commit to do the same here.

Regarding the other suggestions, we might have to agree to disagree on those. Regarding quotes around variables, while I know it's not often _necessary_, I grow weary of `shellcheck` constantly reminding me when I forget to do it, so I've conditioned myself to quote whenever I'm expanding. Likewise, I sometimes forget when some child process is actually going to need one of these variables, so I sometimes use `export` when it's not strictly necessary. Feel free to contact me directly if you feel I must cease & desist from this behavior. :smile:

**Comment by philrz** [comment]
At this point I've incorporated all feedback. Per my opening PR description, at this point merging is blocked on #1716 being implemented.

**Pull request merged**
